src/js/pages/universe/PackagesTab.js

```diff
@@ -30,6 +33,18 @@ const METHODS_TO_BIND = [
   'handleSearchStringChange'
 ];
 
+const shouldRenderUniverseOption = Hooks.applyFilter(
+  'hasCapability', false, 'packageAPI'
+);
+
+if (shouldRenderUniverseOption) {
+  MountService.MountService.registerComponent(
```

Comment:
I'm about to merge it, there's one thing I can't agree with though.

I'd rather not put this code into `src/js/pages/universe/PackagesTab.js`. It's just not the right place. 
Let's create a `universe` dir in the `plugins` with `index.js` and `hooks.js` and we put this code into `hooks.js`
Then all the universe related code will eventually be moved there as well.

//cc @MatApple @orlandohohmeier